ambry-rest/src/main/java/com/github/ambry/rest/PublicAccessLogger.java

```diff
@@ -13,27 +13,33 @@
  */
 package com.github.ambry.rest;
 
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
+//import org.slf4j.Logger;
+//import org.slf4j.LoggerFactory;
+import org.apache.logging.log4j.Logger;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.message.StringMapMessage;
 
 /**
  * Logs requests and responses to public access log.
  */
 public class PublicAccessLogger {
 
-  private static final Logger publicAccessLogger = LoggerFactory.getLogger("PublicAccessLogger");
+  //private static final Logger publicAccessLogger = LoggerFactory.getLogger("PublicAccessLogger");
+  private static final Logger publicAccessLogger = LogManager.getLogger("PublicAccessLogger");
 
   private final String[] requestHeaders;
   private final String[] responseHeaders;
+  private boolean enableStructuredLogging;
 
   /**
    * @param requestHeaders the request headers to log.
    * @param responseHeaders the response headers to log.
+   * @param enableStructuredLogging true if we want to use structured logging (for Kusto), false otherwise
```

Comment:
nit: javadocs formatting
Should be
```
@param enableStructuredLogging {@code true} if we want to use structured logging (for Kusto), {@code false} otherwise
```

Reply:
Thanks, I did not know this.